src/transform/transform.ts

```diff
@@ -39,14 +41,28 @@ async function process(session: Session<CodeModel>) {
   session.model.language.go!.headAsBoolean = headAsBoolean;
   processOperationRequests(session);
   processOperationResponses(session);
+  // parse operation comments that are in Markdown to convert them to plain text
+  for (const group of values(session.model.operationGroups)) {
```

Comment:
Seems like this could be folded into `processOperationRequests()`.

As an aside, we could probably consolidate `processOperationRequests()` and `processOperationResponses()` to only loop once.

Reply:
Done. Parsing the comments on operations in `processOperationRequests`. About joining the processing of operation requests and responses, that's a good improvement. I could do it in a future PR.